pkg/gen/input/workflows/internal/file/create_release_pr.go

```diff
@@ -24,7 +24,7 @@ var createReleasePRTemplate = `# DO NOT EDIT. Generated with:
 #
 name: Create Release PR
 on:
-  create:
+  push:
```

Comment:
Branch filters work only on `push` and `pull_request`, but then we need to figure a way to not create a PR when it's already created.